client/java/src/main/java/com/linecorp/centraldogma/internal/client/AbstractWatcher.java

```diff
@@ -288,7 +288,12 @@ private void notifyListeners() {
 
         final Latest<T> latest = this.latest;
         for (BiConsumer<? super Revision, ? super T> listener : updateListeners) {
-            listener.accept(latest.revision(), latest.value());
+            try {
+                listener.accept(latest.revision(), latest.value());
+            } catch (Exception e) {
+                logger.warn("Exception thrown for watcher ({}/{}{}) on ({}), ",
```

Comment:
nit: For consistency with other logs.
```suggestion
                logger.warn("Exception thrown for watcher ({}/{}{}): rev={}",  ..., latest.revision()
```